maliput_sim/core/components.py

```diff
@@ -27,19 +27,48 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+from typing import List
+
+import maliput_sim
```

Comment:
If here I do 
`from maliput_sim.core.ecm import Entity` instead and in the 43 line I do
`    def set_entity(self, entity: 'Entity'):` then I get a cyclincg dependency

```
ImportError: cannot import name 'Component' from partially initialized module 'maliput_sim.core.components' (most likely due to a circular import) (/home/franco/maliput_ws/install/maliput_sim/lib/python3.8/site-packages/maliput_sim/core/components.py)
```